src/app/app.interceptors.ts

```diff
@@ -68,22 +68,60 @@ export class TokenInterceptor implements HttpInterceptor {
       });
     }
 
-    // for guest users, add a custom header with a unique id
-    if (!this.authSvc.isLoggedIn()) {
-      // Get (or create of needed) the guestUserID in local storage
-      let guestUuid = localStorage.getItem('guestUuid');
+    if (request.url.indexOf(this.envService.apiUrl) > -1) {
+      // Add information about what app is making the request
 
-      if (!guestUuid) {
-        guestUuid = uuidv4();
-        localStorage.setItem('guestUuid', guestUuid);
-      }
-      request = request.clone({
+      // Disabling custom headers due to https://tacc-main.atlassian.net/browse/WG-191
+      // and using additional query params
+      let analytics_params = {};
+
+      /*request = request.clone({
         setHeaders: {
-          'X-Guest-UUID': guestUuid,
+          'X-Geoapi-Application': 'hazmapper',
         },
-      });
+      });*/
+
+      analytics_params = { ...analytics_params, application: 'hazmapper' };
```

Comment:
Need to change application here to 'taggit'

Reply:
😬  thanks 👍